code/game/objects/machinery/miner.dm

```diff
@@ -26,6 +33,8 @@
 	var/miner_integrity = 100
 	///Max health of the miner
 	var/max_miner_integrity = 100
+	///What type of upgrade it has installed
```

Comment:
```suggestion
	///What type of upgrade it has installed. defines [MINER_RESISTANT]
```

Reply:
Also, what does this variable do? include that in the autodoc

Reply:
changed it to be more informative of what it does.